MeetU/MeetU/Scripts/app/meetu-meetup.js

```diff
@@ -92,11 +92,28 @@
                     mview.newComment = "";
                 },
                     function (e) {
-                        console.log(e);
+                        $log.error(e);
                     }
                 );
             };
             //
+            //  Delete comment 
+            //
+            $scope.deleteComment = function (commentId) {
+                var CommentView = $resource('/api/Comments/');
```

Comment:
Since `CommentView` has been created at line 16, this line is no necessary. We can use `CommentView` directly. I guess my experiment misunderstood you somehow.. :laughing: